tests/integration/test_ses.py

```diff
@@ -308,4 +308,17 @@ def test_ses_sns_topic_integration(
         )
 
         messages = sqs_receive_num_messages(sqs_queue, 3)
+        # the messages may arrive out of order so sort
```

Comment:
Thanks for addressing this! I thought about this in the previous PRs, that if the queue is not FIFO, this might happen while testing. Thanks! 👍